### ISSUE TITLE
No flows table in Clickhouse-server

### ISSUE THREAD

**Issue by Yoshihiro-jp** [ntop/ntopng#6335]
**Environment**:
* OS name: [e.g. Ubuntu] "Ubuntu 20.04.4 LTS"
* OS version: [e.g. 18.04] 20.04.4 LTS
* Architecture: [e.g. amd64] x86_64
* ntopng version/revision: [e.g. 5.1.211223] 5.2.220217 r4743

**What happened**:
I did install the ntopng Enterprise L bundle license today.（NFR  License)
Then the ntong.flows table is not created in the clickhouse-server.

[Actual table status]
ntopng :) use ntopng

USE ntopng

Query id: a2481d1b-5f87-4f10-ad3c-e595593f469d

Ok.

0 rows in set. Elapsed: 0.000 sec.

ntopng :) show tables

SHOW TABLES

Query id: c8fea75e-ceb6-491e-8012-95478118727f

lqnameqqqqqqqqqqqqqqqqqqqqqk
x active_monitoring_alerts x
x all_alerts               x
x flow_alerts              x
x host_alerts              x
x interface_alerts         x
x mac_alerts               x
x network_alerts           x
x snmp_alerts              x
x system_alerts            x
x user_alerts              x
mqqqqqqqqqqqqqqqqqqqqqqqqqqj

10 rows in set. Elapsed: 0.001 sec.

ntopng :)

**What did you expect to happen?**
It should create the ntopng.flows table.

**How did you reproduce it?**
Newly install the ntopng.

**Debug Information**:
Kindly refer to the attached log files.
[ntopng.zip](https://github.com/ntop/ntopng/files/8107521/ntopng.zip)

**Comment by MatteoBiscosi** [comment]
Hi @Yoshihiro-jp could you please tell me your ntopng --version and send me ntopng.conf? (if you don't want to share your ntopng configuration, please send it at `biscosi at ntop.org`)

**Comment by Yoshihiro-jp** [comment]
Dear @MatteoBiscosi ,
Thank you for your reply.
Sure I attached the ntopng.conf.
[ntopng_conf.zip](https://github.com/ntop/ntopng/files/8109308/ntopng_conf.zip)

then the version information is below.

root@ntopng:/home/ntop# ntopng --version
Version:        5.2.220217 [Enterprise/Professional build]
GIT rev:        5.2-stable:8f30466104df73e33587748fdc70f89db025f7e6:20220217
Pro rev:        r4743
Built on:       Ubuntu 20.04.3 LTS
System Id:      L85F380D1920CAB22--U85F380D10E070E07--OL
Platform:       x86_64
Edition:        Enterprise L (Bundle)
License Type:   Time-Limited [License Ok]
Validity:       Until Mon Feb 20 05:48:29 2023
License:        KM1d4Ugk00TB4iNMv+9vjt4lXCAAczSGR4RaWRYRvTGBbS4lOQQdqLw7eM5Sf4Wt+G14M
                fV1tI4TcXlMkE1RUfhMd1tHyaBvWFntmFgHw4GYgQg3t/7XFYDzrE77TS+6Htx1mXCOqK
                Eyd3MZKIuSwUIVXyj4DB0o0BGHEWJ0O2GphtDPtfvMg1uIUXP768kEl9+1V9lh26jgC3f
                j/Ah9TF0FdkgJxgKEJlDpnZEOpL7GPZ3kVmkBOSfzPpRVG+mCj+2Xr9wrSWg19HK3mRy+
                txh1cVSRKmA6tjWjbKhfJYjJxKX30GwbL4b2p5c5+K7fo2jmQhEPnbP3aq1h2y5nAw==
License Hash:   21FDFA771F8A4F5B617890524DD639C516768721092FD694B3-N
root@ntopng:/home/ntop#


regards,
Yoshihiro

**Comment by pwchi** [comment]
I have same issue.
Newly install the ntopng and useing clickhouse have problem.
If migration from nIndex is fine.

```
pwchi@ntop:/etc/ntopng/ntopng.conf.d$ cat my.conf 
#指定監聽界面
--interface=ens192

#Disable http
--http-port=0

#Enable https
--https-port=443

# 啟用 Clickhouse flow dump
-F="clickhouse;127.0.0.1;ntopng;flows;default;"
```

```
Version:	5.2.220221 [Enterprise/Professional build]
GIT rev:	5.2-stable:f384143889c2389b3c9ec16b1599dd815bdcc582:20220221
Pro rev:	r4744
Built on:	Ubuntu 20.04.3 LTS
System Id:	L689D67B1550561EF--U689D67B1B826C20A--OL
Platform:	x86_64
Edition:	Enterprise L
```

```
22/Feb/2022 01:19:41 [ClickHouseImport.cpp:91] WARNING Received exception from server (version 22.2.2): Code: 60. DB::Exception: Received from 127.0.0.1:9000. DB::Exception: Table ntopng.flows doesn't exist. (UNKNOWN_TABLE) (query: INSERT INTO flows (FLOW_ID,IP_PROTOCOL_VERSION,FIRST_SEEN,LAST_SEEN,VLAN_ID,PACKETS,TOTAL_BYTES,SRC2DST_BYTES,DST2SRC_BYTES,SRC2DST_DSCP,DST2SRC_DSCP,PROTOCOL,IPV4_SRC_ADDR,IPV6_SRC_ADDR,IP_SRC_PORT,IPV4_DST_ADDR,IPV6_DST_ADDR,IP_DST_PORT,L7_PROTO,L7_PROTO_MASTER,L7_CATEGORY,FLOW_RISK,INFO,PROFILE,NTOPNG_INSTANCE_NAME,INTERFACE_ID,STATUS,SRC_COUNTRY_CODE,DST_COUNTRY_CODE,SRC_LABEL,DST_LABEL,SRC_MAC,DST_MAC,COMMUNITY_ID,SRC_ASN,DST_ASN,PROBE_IP,OBSERVATION_POINT_ID,SRC2DST_TCP_FLAGS,DST2SRC_TCP_FLAGS,SCORE,CLIENT_NW_LATENCY_US,SERVER_NW_LATENCY_US,CLIENT_LOCATION,SERVER_LOCATION,SRC_NETWORK_ID,DST_NETWORK_ID,INPUT_SNMP,OUTPUT_SNMP,SRC_HOST_POOL_ID,DST_HOST_POOL_ID) FORMAT CSV)
```

```
22/Feb/2022 01:21:24 Table ntopng.flows doesn't exist [183233818,183666116,342933711,342861122,342894379,348365424,348373225,351177337,351167989,362738746,362820633,409369103,409378401,411149641,411139136,139650122978825,139650122089107]
```

**Comment by MatteoBiscosi** [comment]
Hi, I was able to reproduce the issue, thanks a lot for your report, we are already working on it

**Comment by lucaderi** [comment]
Issue fixed in ddceccdaaf2a8b22b30a692bad2e55aa80167fc3. New packages will be available within a couple of hours. Thanks for reporting it.

**Comment by Yoshihiro-jp** [comment]
Dear @lucaderi and @MatteoBiscosi ,

Thank you I found that the stable version was fixed but the nightly build one is not yet.
I hope that it would be fixed too.


ntopng :) show tables

SHOW TABLES

Query id: 6d997f77-2ffc-42f4-8a60-05c241637a63

lqnameqqqqqqqqqqqqqqqqqqqqqk
x active_monitoring_alerts x
x all_alerts               x
x flow_alerts              x
x host_alerts              x
x interface_alerts         x
x mac_alerts               x
x network_alerts           x
x snmp_alerts              x
x system_alerts            x
x user_alerts              x
mqqqqqqqqqqqqqqqqqqqqqqqqqqj

10 rows in set. Elapsed: 0.001 sec.

ntopng :) q
Bye.
ntop@ntopng:~$ sudo ntopng -V
4Version:       5.3.220223 [Enterprise/Professional build]
GIT rev:        dev:dcd23241c77d501373f153902d37c5a1037f246f:20220223
Pro rev:        r4784
Built on:       Ubuntu 20.04.3 LTS
System Id:      L3BB3134B920CAB22--U3BB3134BA7F60C83--OL
Platform:       x86_64
Edition:        Enterprise L
License Type:   Time-Limited [License Ok]
Validity:       Until Tue Feb 21 09:02:51 2023
License:        HD4XQFEf1bD0nLWIOyBKO5d23Ry1xzWab1OsLRu13BiizkcG+b4glLD7r2HWQsOsOAmxJ
                XNgpEXPedHN00jIAn6M900WU8ClkIe2Sz3tk1rdNP3Pj7wQjI1chmLe5sFYYispyRkwBq
                lyn/1Z81LOGPPRGdZMo1wJQj1L7JEIg2Re+gxJuz5lY1cT8VpLubGaXUejZZK6DxkmKfh
                z4ACQ4vjrHBasAivy1xuT9QEregc+VbyE1tE7GeocIYk9ktjFbR/Rbnlt35d4WZmuk/Fo
                z239mNf1D/CSkjeebs2slrSeP7D5dSsFTfkUahNDXtlKJh42q2wg7CDy+Q6e1XfSuQ==
License Hash:   61F8BB769B221A465EEADA20BD9FE76D16769377718816E01B-N

regards,
Yoshihiro
